contribs/gmf/src/directives/print.js

```diff
@@ -985,16 +995,18 @@ gmf.PrintController.prototype.getLegend_ = function(scale) {
   // For each visible layer in reverse order, get the legend url.
   layers.reverse().forEach((layer) => {
     classes = [];
-
     if (layer.getVisible() && layer.getSource()) {
       // For WMTS layers.
       if (layer instanceof ol.layer.Tile) {
-        layerName = layer.get('layerNodeName');
-        icons = this.ngeoLayerHelper_.getWMTSLegendURL(layer);
+        layerName = `${layer.get('layerNodeName')}`;
```

Comment:
Why this change?

Reply:
Because closure-compiler want a string below in some function. And the "get" here can potentially give me other things.

Reply:
ok :-)